### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
added avatar component 

- [code sandbox] (https://1zj1uh.csb.app/components/avatar/avatar.html)

### CODE REVIEW

**Review by poojakelkar — APPROVED** [review]
Looks good to merge

**Review by Rajitha04 — CHANGES_REQUESTED** [review]
Screenshot_2022-02-20-20-16-49-92_40deb401b9ffe8e1df2f1cc5ba480b12
Nav bar and footer seems to be not aligned properly , rest looks good!

**Review by Rajitha04 — APPROVED** [review]
Looks good to merge!👍

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+}
+.size1{
+    font-size: 25px;
+}
+.size2{
+    font-size: 23px;
+}
+.size3{
+    font-size: 18px;
+}
```
Comment (anupkrbid) [inline components/avatar/avatar.css:34]: you should use rems instead of px as much as possible.

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        <div><a href="#">Toast</a></div>
+    </aside>
+    <div class="main">
+    <h1 >Avatar</h1>
+    <p class="size3">Avatar represents a image in a circular Container</p>
+    <img class="avatar" src="https://miro.medium.com/max/1193/1*NVU5-vgxMwufQAsXfr67-Q@2x.jpeg" alt="Pic">
+    <br>
+    <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=31px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=118%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522https%253A%252F%252Fmiro.medium.com%252Fmax%252F1193%252F1*NVU5-vgxMwufQAsXfr67-Q%25402x.jpeg%2522%2520alt%253D%2522Pic%2522%253E%250A"
+    style="width: 915px; height: 90px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline components/avatar/avatar.html:40]: avoid using inline styles. you can move this to your css file and then use the class.

File: components/avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        <h2>Components</h2>
+        <div><a href="#">Alert</a></div>
+        <div><a href="#">Avatar</a></div>
+        <div><a href="#">Badge</a></div>
+        <div><a href="#">Button</a></div>
+        <div><a href="#">Card</a></div>
+        <div><a href="#">Grid</a></div>
+        <div><a href="#">Image</a></div>
+        <div><a href="#">Input</a></div>
+        <div><a href="#">Toast</a></div>
```
Comment (anupkrbid) [inline components/avatar/avatar.html:31]: you can use a list to represent these instead of using divs

**Review by anupkrbid — COMMENTED** [review]
overall good. 
The comments that I have added are the same for other similar cases.

**Pull request merged**